<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactView</class>
 <widget class="QWidget" name="ContactView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QTabWidget" name="contact_pages">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <widget class="QWidget" name="info_page">
      <attribute name="icon">
       <iconset resource="../Keyhotee.qrc">
        <normaloff>:/images/info_user.png</normaloff>:/images/info_user.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <widget class="QStackedWidget" name="info_stack">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>-10</y>
         <width>635</width>
         <height>326</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="info_status">
        <widget class="QWidget" name="widget_4" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>441</width>
           <height>91</height>
          </rect>
         </property>
         <widget class="QLabel" name="security_level_view">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>9</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Blocked</string>
          </property>
         </widget>
         <widget class="QLabel" name="phone_prefix">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>28</y>
            <width>34</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Phone:</string>
          </property>
         </widget>
         <widget class="QLabel" name="phone_view">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>28</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="email_prefix">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>47</y>
            <width>28</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
         <widget class="QLabel" name="email_view">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>47</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="security_level_prefix">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Security Level:</string>
          </property>
         </widget>
         <widget class="QLabel" name="public_key_prefix">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>66</y>
            <width>52</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Public Key:</string>
          </property>
         </widget>
         <widget class="QLabel" name="public_key_view">
          <property name="geometry">
           <rect>
            <x>91</x>
            <y>66</y>
            <width>210</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Unknown</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
         <widget class="QToolButton" name="public_key_to_clipboard">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>64</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>Copy to clipboard</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../Keyhotee.qrc">
            <normaloff>:/images/win/editcopy.png</normaloff>:/images/win/editcopy.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="notes_group">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>429</width>
           <height>161</height>
          </rect>
         </property>
         <property name="title">
          <string>Notes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="notes_display">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="info_edit">
        <widget class="QGroupBox" name="notes_groupbox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>190</y>
           <width>411</width>
           <height>80</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>80</height>
          </size>
         </property>
         <property name="title">
          <string>Notes</string>
         </property>
         <widget class="QPlainTextEdit" name="notes">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>391</width>
            <height>51</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>276</y>
           <width>617</width>
           <height>41</height>
          </rect>
         </property>
         <widget class="QPushButton" name="cancel_button">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
         <widget class="QPushButton" name="save_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="geometry">
          <rect>
           <x>9</x>
           <y>9</y>
           <width>617</width>
           <height>31</height>
          </rect>
         </property>
         <widget class="QLineEdit" name="firstname">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>First Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lastname">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>0</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Last Name</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="id_status">
         <property name="geometry">
          <rect>
           <x>320</x>
           <y>46</y>
           <width>200</width>
           <height>16</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>ID Status</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="id_edit">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>46</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Keyhotee ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="privacy_level_label">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>111</y>
           <width>63</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Privacy Level</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>privacy_comboBox</cstring>
         </property>
        </widget>
        <widget class="QComboBox" name="privacy_comboBox">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>111</y>
           <width>120</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Blocked</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="No one should know you have ever communicated with this individual.">Hidden Relationship</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="The government knows you are in communication with this individual.">Public Relationship</string>
          </property>
         </item>
        </widget>
        <widget class="QLabel" name="email_label">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>137</y>
           <width>24</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Email</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>email</cstring>
         </property>
        </widget>
        <widget class="QLineEdit" name="email">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>137</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="phone_label">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>163</y>
           <width>30</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Phone</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>phone</cstring>
         </property>
        </widget>
        <widget class="QLineEdit" name="phone">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>163</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="public_key_label">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>85</y>
           <width>48</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Public Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>public_key</cstring>
         </property>
        </widget>
        <widget class="QLineEdit" name="public_key">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>85</y>
           <width>133</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <zorder>privacy_comboBox</zorder>
        <zorder>email_label</zorder>
        <zorder>email</zorder>
        <zorder>phone_label</zorder>
        <zorder>phone</zorder>
        <zorder>public_key_label</zorder>
        <zorder>public_key</zorder>
        <zorder>notes_groupbox</zorder>
        <zorder>widget_5</zorder>
        <zorder>widget_6</zorder>
        <zorder>id_status</zorder>
        <zorder>id_edit</zorder>
        <zorder>privacy_level_label</zorder>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="chat_page">
      <attribute name="icon">
       <iconset resource="../Keyhotee.qrc">
        <normaloff>:/images/chat.png</normaloff>:/images/chat.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Communication</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="opaqueResize">
          <bool>true</bool>
         </property>
         <property name="handleWidth">
          <number>5</number>
         </property>
         <widget class="QTextBrowser" name="chat_conversation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>10</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="chat_input"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QWidget" name="toolbar_container" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget { background=qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255)) }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Keyhotee.qrc"/>
 </resources>
 <connections/>
</ui>
